internal/cmd/gocdk/init.go

```diff
@@ -81,6 +82,16 @@ func init_(ctx context.Context, pctx *processContext, args []string) error {
 	if err := materializeTemplateDir(projectDir, "init", tmplValues); err != nil {
 		return xerrors.Errorf("gocdk init: %w", err)
 	}
+	fmt.Fprintf(pctx.stdout, "Project created at %s with:\n", projectDir)
+	fmt.Fprintln(pctx.stdout, "- Go HTTP server")
+	fmt.Fprintln(pctx.stdout, "- Dockerfile")
+	fmt.Fprintln(pctx.stdout, "- 'dev' biome for local development settings")
+	fmt.Fprintln(pctx.stdout)
+	fmt.Fprintf(pctx.stdout, "Run `cd %s`, then run:\n", f.Arg(0))
+	fmt.Fprintln(pctx.stdout, "- `gocdk serve` to develop locally")
```

Comment:
Let's try and say something here about the live code reloading? (The `serve` output itself should also probably say something but if I'm a new user who read the docs sloppily or not at all I might not understand the core value prop of `serve`)

Reply:
Done.